app/uk/gov/hmrc/thirdpartyapplication/models/JsonFormatters.scala

```diff
@@ -55,6 +67,7 @@ trait JsonFormatters extends UtcMillisDateTimeFormatters {
   implicit val formatRejectUpliftRequest = Json.format[RejectUpliftRequest]
   implicit val formatResendVerificationRequest = Json.format[ResendVerificationRequest]
   implicit val formatAddCollaboratorRequest = Json.format[AddCollaboratorRequest]
+  implicit val formatAddTermsOfUseAgreement = Json.format[AddTermsOfUseAcceptanceRequest]
```

Comment:
TermsOfUseAcceptance but field is TermsOfUseAgreement (which is something else)

Reply:
fixed